src/test/scala/centaur/FinalDirsSpec.scala

```diff
@@ -0,0 +1,32 @@
+package centaur
+
+import java.nio.file.{Paths, Path}
+
+import cats.data.Validated.{Invalid, Valid}
+import centaur.test.formulas.TestFormulas
+import centaur.test.workflow.Workflow
+import org.scalatest.{FlatSpec, Matchers, ParallelTestExecution}
+
+object FinalDirsSpec {
+  val FinalDir = Paths.get("src/main/resources/finalCopy")
+  val OutputsDirTest = FinalDir.resolve("final_workflow_outputs.test")
+  val LogDirTest = FinalDir.resolve("final_workflow_log_dir.test")
+  val CallLogsDirTest = FinalDir.resolve("final_call_logs_dir.test")
+}
+
+class FinalDirsSpec extends FlatSpec with Matchers with ParallelTestExecution {
+  import FinalDirsSpec._
+
+  "finalOutputs" should "place files in output dir" in test(OutputsDirTest, "final_workflow_outputs_dir")
```

Comment:
It'd be nice if the name of `test` could be a bit more descriptive. For a little while I thought this was some weird scalatest built-in that I'd never seen before